client/src/app/components/CreateProject/styles.css

```diff
@@ -0,0 +1,65 @@
+.form-wrapper {
+  position: absolute;
+  left: 25%;
+  top: 20%;
+  margin: 15px;
+  padding: 10px 25px;
+  border: 1px solid #d6d6d6;
+  box-shadow: 0 2px 8px 1px #d6d6d6;
+  width: 50%;
+  background: white;
+  z-index: 10;
+}
+
+.form-wrapper h2 {
+  padding: 0;
+  margin-bottom: 0;
+  text-align: left;
+  font-size: 20px;
+  padding-bottom: 15px;
+  color: #00576b;
+  text-transform: uppercase;
+}
+
+.form-item {
+  display: flex;
+  flex-direction: column;
+  padding-top: 15px;
+  margin: 5px;
+}
+
+.form-wrapper .form-item .title span {
```

Comment:
So many levels of CSS class selectors could be avoided. Most often, selectors with one class is enough. I think here you wanted to be more specific and say which '.title' class you want to change.

This kind of issue can be avoided with a CSS naming system like BEM. But the easiest these days is to use CSS-modules (already used in the app). With css-modules, you get unique class names, thus avoiding class name collision when using a general class name such as 'title'.